### PR TITLE
Function Closures in Java Script

### PR DESCRIPTION
**Pull request by smorampudi-nisum-com**

### CODE REVIEW

File: Week 1 Assignment/privateMethodsWithClosure.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Closure Update Application</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+<script>
```
Comment (vllmstl) [inline Week 1 Assignment/privateMethodsWithClosure.html:10]: Always create a new file to store scripts.
Move the below code to a new file and link it on this page.

File: Week 1 Assignment/ClosuresInLoops.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+<html lang="en">
+<head>
+  <title>Closure In Loops Application</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mootools/1.6.0/mootools-core-compat.js"></script>
+  <style>
```
Comment (vllmstl) [inline Week 1 Assignment/ClosuresInLoops.html:11]: Always create a new file to store css.
Move the below code to a new file and link it on this page.

File: Week 1 Assignment/ClosureUpdate.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Closure Update Application</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+<script>
```
Comment (vllmstl) [inline Week 1 Assignment/ClosureUpdate.html:10]: Always create a new file.
Move the below code to a new file and link it on this page.

File: Week 1 Assignment/BasicClosure.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Basic Closure Application</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+<script>
```
Comment (vllmstl) [inline Week 1 Assignment/BasicClosure.html:10]: Always create a new file.
Move the below code to a new file and link it on this page.

**Review by vllmstl — COMMENTED** [review]
+ bootstrap
+ Lexical scope explained
- separation of concerns (Move content to it's own files)
- Good/Bad use of closures with examples

File: Week 1 Assignment/ClosureUpdate.html (hunk 3 above)
Comment (smorampudi-nisum-com) [reply Week 1 Assignment/ClosureUpdate.html:10]: Sure karthik, will Redo and Upload the files again

File: Week 1 Assignment/ClosuresInLoops.html (hunk 2 above)
Comment (smorampudi-nisum-com) [reply Week 1 Assignment/ClosuresInLoops.html:11]: Sure karthik, will Redo and Upload the files again

File: Week 1 Assignment/privateMethodsWithClosure.html (hunk 1 above)
Comment (smorampudi-nisum-com) [reply Week 1 Assignment/privateMethodsWithClosure.html:10]: Sure karthik, will Redo and Upload the files again

**Comment by vllmstl** [comment]
No need to close a Pull Request, you can make changes and pull to the same branch. It will reflect here.

**Pull request merged**